components/Search/Search.tsx

```diff
@@ -1,109 +1,148 @@
-import { useEffect, ComponentProps } from "react";
+import React, { useEffect, useRef, createElement, Fragment } from "react";
+import { autocomplete } from "@algolia/autocomplete-js";
+import "@algolia/autocomplete-theme-classic";
+import { render } from "react-dom";
 import styled from "styled-components";
-import { css, wrapper, StyledSystemWrapperProps } from "components/system";
-import Icon from "components/Icon";
+import css from "@styled-system/css";
+import Box from "components/Box";
+import { debounced } from "server/search-api-helpers";
 import { SearchStyles } from "./SearchStyles";
-import docVersions from "../../config.json";
 
-export interface SearchProps {
-  id?: string;
-  version?: string;
-}
+export const getSearchResults = async (query: string) => {
+  let href = "";
+  if (typeof window !== "undefined") {
+    href = window.location.href;
+  }
+
+  try {
+    const rawResponse = await fetch(
+      `/docs/api/search/?query=${query}&url=${encodeURIComponent(href)}`,
+      {
+        method: "GET",
+      }
+    );
+
+    const response = await rawResponse.json();
+    return response.map((res) => ({ ...res, label: res.title }));
+  } catch (e) {
+    throw e;
```

Comment:
Let's show error in the UI instead of throwing error here.

Reply:
under normal conditions, an error should not happen here. So I think it makes no sense to show the error in the UI. I've written console.error()